### PR TITLE
Bugfix: Lacks of a Content-Security-Policy (CSP) in lisk-desktop

### LINKED ISSUE #5111: Lacks of a Content-Security-Policy (CSP) in lisk-desktop
### Description

Define a CSP in the application index.html page using the meta tag such as: `<meta http-equiv="Content-Security-Policy" content="default-src 'none'">`

The lisk-desktop application renders its content without a Content-Security-Policy (CSP). A CSP adds extra protection against cross-site scripting (XSS) and data injection by allowing developers to specify which source the browser can execute or render code from.

### Reference

https://www.electronjs.org/docs/latest/tutorial/security#7-define-a-content-security-policy

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5111

### How was it solved?
- Set content security policy

### How was it tested?

- Manually

### CODE REVIEW

**Review by oskarleonard — DISMISSED** [review]

**Review by ManuGowda — COMMENTED** [review]
Devnet its not working 
Screenshot 2023-07-05 at 12 02 45 PM

File: setup/react/index.html [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
+  <head>
+    <base href="./">
+    <meta charset="utf-8">
+    <meta name="application-name" content="lisk-desktop-<%= htmlWebpackPlugin.options.VERSION %>" data-lisk-desktop-version="<%= htmlWebpackPlugin.options.VERSION %>">
+    <meta name="viewport" content="width=device-width">
+    <meta name="keywords" content="Lisk, Lisk, wallet">
+    <meta property="og:title" content="Lisk">
+    <meta property="og:description" name="description" content="Lisk is an all-in-one solution to manage your Lisk ID, access and send LSK tokens, as well as stake for validators.">
+    <meta property="og:image" content="https://lisk.com/hub/assets/images/og-logo.png">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com http://devnet-service.liskdev.net wss://*.walletconnect.com">
```
Comment (ManuGowda) [inline setup/react/index.html]:
```suggestion
    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/ data: https://api.mapbox.com; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com http://devnet-service.liskdev.net wss://*.walletconnect.com">
```

Comment (ikem-legend) [reply setup/react/index.html]: Fixed

File: setup/react/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
     <meta property="og:image" content="https://lisk.com/hub/assets/images/og-logo.png">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com http://devnet-service.liskdev.net wss://*.walletconnect.com">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/ https://unpkg.com/leaflet@1.9.4/; img-src 'self' data: https://raw.githubusercontent.com/LiskHQ/app-registry/main/ https://api.mapbox.com/; font-src 'self' data: https://fonts.gstatic.com/; script-src 'self' 'unsafe-inline' 'unsafe-eval'; connect-src 'self' https://*.lisk.com wss://*.lisk.com http://devnet-service.liskdev.net:9901 ws://devnet-service.liskdev.net:9901 wss://*.walletconnect.com">
```
Comment (ManuGowda) [inline setup/react/index.html:12]: what about alphanet? can we try more generic `connect-src` approach?

Comment (ikem-legend) [reply setup/react/index.html:12]: While I can hardcode alphanet, testnet and mainnet or use a pattern, we still need to figure out the rest of the URLs

**Review by oskarleonard — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

File: setup/react/index.html (hunk 2 above)
Comment (ManuGowda) [reply setup/react/index.html:12]: for now there is no clear direction on how to make it generic, hence approving and proceeding further. We can seek for improvements later.

**Pull request merged**